keyboards/minidox/keymaps/bepo/keymap.c

```diff
@@ -0,0 +1,111 @@
+#include QMK_KEYBOARD_H
+#include "keymap_bepo.h"
+
+extern keymap_config_t keymap_config;
+
+#define TAPPING_TERM 450
+
+#define _BEPO 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
It may be best to use an enum here.  It's easier to manage, long term. 

```suggestion
enum layers {
    _BEPO,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```